README.md

```diff
@@ -8,6 +8,7 @@
 
 ## Authors
 
+* [Brian Ginsburg](https://github.com/bgins), [Fission](https://fission.codes)
```

Comment:
Wasn't sure how the authors are ordered here, but it looks like alphabetical order? If anything, I should go last because this contribution to the spec is quite small. 🙂

Reply:
I'm totally fine with this, and authors vs contributor is not _super_ clear, but a heads up: having your name in this part of the spec may mean that you'll get pinged with questions about the design etc. If you're cool with that, then so am I :) Otherwise we can add you to the acknowledgements.

Reply:
I think I would prefer to be mentioned in the acknowledgements. I could respond to many questions about the spec, but probably not design questions. Best to leave that to the authors that have contributed more substantially.

What would be a good way to add an acknowledgement? Should we add a section for that or maybe it is implicit in `Notices.md`?

Reply:
Oh geez, do we not have one now? 😱